cirq-google/cirq_google/line/placement/greedy.py

```diff
@@ -24,6 +24,7 @@
 
 if TYPE_CHECKING:
     from cirq_google.line.placement.sequence import LineSequence
+    import cirq
```

Comment:
Looks like this is not needed; I don't see any new usage of `cirq` in type annotations.